doc/release-notes-1.11.rst

```diff
@@ -240,7 +240,7 @@ release:
 * :github:`3322` - [IPv6 TAHI] Section 3: RFC 4862 - IPv6 Stateless Address Autoconfiguration
 * :github:`3323` - [IPv6 TAHI] Section 2: RFC 4861 - Neighbor Discovery for IPv6
 * :github:`3329` - Build warnings [-Wpointer-sign] with LLVM/icx (bluetooth_ipsp)
-* :github:`3345` - Missing board documentation for riscv32/zedboard_pulpino
+* :github:`3345` - Missing board documentation for riscv32/
```

Comment:
Deleting release notes history doesn't seem right. The github issue 3345 remains, so should this release note.

Reply:
addressed. was included by mistake.